formbricks/formbricks
# feat: auto subscribing to teams survey responses email

## What does this PR do?
- Subscribe Team members to new surveys created in their team by anybody (unless explicitly disabled)
- Send email when a new response is received
- The email has 2 options for the users to enable these emails
    - Disable this survey's responses
    - Disable auto-subscribing to this team's further created surveys
- The email link is built such that the moment the user clicks on it, they are taken to the notification page and the trigger to disable that survey/team is triggered in front of a user followed by a toast.
- Use updateUser service to update user rather than using prisma  client

## How should this be tested?

- Create new surveys, send responses, they should be in your inbox
- Invite a team member, see that they are auto subscribed by sending a response and validating that both of you receive a response email
- Try clicking on email disable links both in authenticated as well as unauthenticated state

## Checklist



### Required


### Appreciated
